content/updates/terraform-controller-overview.md

```diff
@@ -0,0 +1,237 @@
+
+### Utilizing Other Optional Features
+
+We're also interested in the (Branch Planner feature)[(https://weaveworks.github.io/tf-controller/branch-planner/branch-planner-getting-started/)]. This new component detects an open PR and either creates a new Terraform object or updates an existing one, applying a Plan Only mode based on the original Terraform object. This functionality aligns really well with GitOps workflows, providing a dynamic and transparent way to manage infrastructure changes via pull requests.
+
+### Open Source UI
+
+Another area of interest that we'd like to see built out further is the Open Source UI, as outlined in the [Weave GitOps documentation](https://docs.gitops.weave.works/docs/open-source/getting-started/ui-OSS/). The ability to easily oversee and control all the details and stages of infrastructure deployment and management is always a valuable addition.
+
+
+## Time-sensitive: Internal Weaveworks Issues and Further Impact on the Project
+
+While we've been working on this blog post, some quite concerning news has spread in the Weaveworks community. A recent [GitHub comment](https://github.com/weaveworks/tf-controller/issues/1166#issuecomment-1904892837) sheds some light on the fact that Weaveworks, the legal entity behind the project, is facing undisclosed issues that are impacting the direction of the project. We have to admit that the future of the Terraform Controller project seems uncertain, and the future steps of the repository's owning organization are not yet clear. It's hard to know what to expect when there is no official public statement from Weaveworks about the situation.
+
+Despite these challenges, we still hope for the best outcome. The project is still alive, with some team members continuing their work and planning future developments. It's worth noting that the project could either be renamed and retain all repositories under the current organization or move to a different organization.
+
+We're happy to hear that the community is committed to continuing discussions about the project's future and development, and we're ready to support them as believe this tool has some real potential.
+
+## Conclusion
+
+The Terraform Controller could be a game-changer in managing infrastructure as code via GitOps a breeze. Still, we acknowledge that building a solid and usable solution requires further investment. If you're interested in this approach, we welcome any engineering orgs who are interested in using this tool to [reach out and chat with us](https://masterpoint.io/contact/) as we'd love to build a true GitOps Terraform system for you!
```

Comment:
```suggestion
The Terraform Controller could be a real game-changer in managing infrastructure as code via GitOps. We believe with some further investment and improvements that it would be solid competitor to the primary open-source Terraform automation tool, [Atlantis](https://www.runatlantis.io/). Still, we acknowledge that building a strong system around this tool requires effort and our suggestion is to only entertain using this tool if your org is ready for some level of bleeding-edge investment themselves. If you're intrigued by that idea, we welcome any engineering orgs who are eager to operationalize this tool internally to [reach out and chat with us](https://masterpoint.io/contact/) as we'd love to help you build a true GitOps Terraform system!
```